### PR TITLE
General accessibility enhancement (mainly focus management) and introduce InlineInput component

### PR DESCRIPTION
**Pull request by JBWatenbergScality**
**Component**:

InlineInput, Navbar, DropDown

**Description**:

Accessibility enhancement and introduction of the new InlineInput component 

https://github.com/scality/core-ui/assets/75977494/b2621e2e-3a45-4bd0-9192-4405ec5f23e9

### CODE REVIEW

**Comment by Cuervino** [comment]
- From the video demo, the Cancel button of the modal should be the "Outline" style, not "Secondary".
- Is it underlined with a line, or with a dotted line?

File: src/lib/components/dropdown/Dropdown.component.tsx [hunk 1]
```diff
@@ -81,25 +59,36 @@ const DropdownMenuItemStyled = styled.li`
...
   font-size: ${fontSize.base};
+  ${(props) => {
+    console.log(props.isSelected);
+    return props.isSelected
+      ? `background-color: ${props.theme.highlight};`
+      : `background-color: ${props.theme.backgroundLevel1};`;
+  }}
+
+  color: ${getThemePropSelector('textPrimary')};
+  border-top: 0.3px solid ${getThemePropSelector('border')};
```
Comment (Cuervino) [inline src/lib/components/dropdown/Dropdown.component.tsx:70]: Should we make this value of "0.3px" a variable/token, to have consistency across components?

Comment (hervedombya) [reply src/lib/components/dropdown/Dropdown.component.tsx:70]: I think this value is specific to the dropdown component, not sure it's really useful, is it?

**Review by hervedombya — APPROVED** [review]

File: src/lib/components/buttonv2/Buttonv2.component.tsx [hunk 2]
```diff
@@ -97,8 +102,7 @@ export const ButtonStyled = styled.button<Props>`
             border: ${spacing.r1} solid ${brand.infoPrimary};
           }
           &:focus-visible:enabled {
-            outline: dashed ${spacing.r2} ${brand.selectedActive};
-            outline-offset: ${spacing.r2};
+            ${FocusVisibleStyle}s
```
Comment (JBWatenbergScality) [inline src/lib/components/buttonv2/Buttonv2.component.tsx]:
```suggestion
            ${FocusVisibleStyle}
```

**Pull request merged**